app/views/courses/index.html.erb

```diff
@@ -1,100 +1,26 @@
 <%= title('Courses') %>
 
-<div class="CoursesController">
+<div class="CoursesController col-lg-10 offset-lg-1">
   <div class="container">
     <div class="heading-box">
       <div class="heading">
-        <h1>This is Your Path to Learning Web Development</h1>
-      </div>
-      <div class="subheading">
+        <h1 class="text-center camel light curriculum-title">Curriculum</h1>
+      </div> <!-- /.heading -->
+
+      <div>
         <p>
```

Comment:
ah I see 😄 

I would make it look as close to the design as possible, We will probably be updating the copy on this page as well. 

It looks there are no "see more" links in the redesign mock up so I would take them out completley.